<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1218</width>
    <height>782</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget_all" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1211</width>
      <height>751</height>
     </rect>
    </property>
    <widget class="QWidget" name="widget_main" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1191</width>
       <height>541</height>
      </rect>
     </property>
     <widget class="QPushButton" name="addbut">
      <property name="geometry">
       <rect>
        <x>1080</x>
        <y>50</y>
        <width>92</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>添加学生</string>
      </property>
     </widget>
     <widget class="QPushButton" name="openfilebut">
      <property name="geometry">
       <rect>
        <x>1080</x>
        <y>10</y>
        <width>92</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>打开文件</string>
      </property>
     </widget>
     <widget class="QPushButton" name="rankbut">
      <property name="geometry">
       <rect>
        <x>1080</x>
        <y>180</y>
        <width>92</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>总分排序</string>
      </property>
     </widget>
     <widget class="QPushButton" name="findbut">
      <property name="geometry">
       <rect>
        <x>1080</x>
        <y>90</y>
        <width>92</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>删除菜单</string>
      </property>
     </widget>
     <widget class="QPushButton" name="analysisbut">
      <property name="geometry">
       <rect>
        <x>1080</x>
        <y>220</y>
        <width>92</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>统计</string>
      </property>
     </widget>
     <widget class="QPushButton" name="exit">
      <property name="geometry">
       <rect>
        <x>1080</x>
        <y>270</y>
        <width>92</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>退出</string>
      </property>
     </widget>
     <widget class="QTreeView" name="treeView">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1011</width>
        <height>511</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="findbut_2">
      <property name="geometry">
       <rect>
        <x>1080</x>
        <y>130</y>
        <width>92</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>查询</string>
      </property>
     </widget>
     <zorder>treeView</zorder>
     <zorder>addbut</zorder>
     <zorder>openfilebut</zorder>
     <zorder>rankbut</zorder>
     <zorder>findbut</zorder>
     <zorder>analysisbut</zorder>
     <zorder>exit</zorder>
     <zorder>findbut_2</zorder>
    </widget>
    <widget class="QWidget" name="widget_find1" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>500</y>
       <width>431</width>
       <height>201</height>
      </rect>
     </property>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>371</width>
        <height>81</height>
       </rect>
      </property>
      <property name="title">
       <string>查询方式1</string>
      </property>
      <widget class="QPushButton" name="find_id">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>按学号查找</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>50</y>
         <width>181</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>361</width>
        <height>81</height>
       </rect>
      </property>
      <property name="title">
       <string>查询方式2</string>
      </property>
      <widget class="QPushButton" name="find_name">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>按姓名查找</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_2">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>30</y>
         <width>171</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="widget_add" native="true">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>80</y>
       <width>471</width>
       <height>451</height>
      </rect>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>学号：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="nameedit">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>80</y>
        <width>113</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="englishedit">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>250</y>
        <width>113</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>140</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>数学：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="idedit">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>30</y>
        <width>113</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="mathedit">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>140</y>
        <width>113</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>姓名：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>190</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>物理：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="physicsedit">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>190</y>
        <width>113</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>250</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>英语：</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>350</y>
        <width>92</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>确认</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="widget_de" native="true">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>530</y>
       <width>141</width>
       <height>161</height>
      </rect>
     </property>
     <widget class="QPushButton" name="deletebut">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>111</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>删除学生</string>
      </property>
     </widget>
     <widget class="QPushButton" name="back">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>111</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>返回主界面</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="widget_find2" native="true">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>520</y>
       <width>361</width>
       <height>181</height>
      </rect>
     </property>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>351</width>
        <height>121</height>
       </rect>
      </property>
      <property name="title">
       <string>查询方式3:</string>
      </property>
      <widget class="QComboBox" name="searchchoice">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>89</width>
         <height>27</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>30</y>
         <width>21</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>从</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="score_min">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>30</y>
         <width>31</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>30</y>
         <width>21</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>到</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="score_max">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>30</y>
         <width>41</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>30</y>
         <width>31</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>分</string>
       </property>
      </widget>
      <widget class="QPushButton" name="searchmul">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>70</y>
         <width>101</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>按成绩查找</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>30</y>
         <width>69</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>(请填数字）</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="back_2">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>140</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>返回主界面</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="widget_gaojichazhao" native="true">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>20</y>
       <width>1041</width>
       <height>511</height>
      </rect>
     </property>
     <widget class="QTextEdit" name="textEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>-10</y>
        <width>1021</width>
        <height>511</height>
       </rect>
      </property>
     </widget>
    </widget>
    <zorder>widget_add</zorder>
    <zorder>widget_main</zorder>
    <zorder>widget_find1</zorder>
    <zorder>widget_de</zorder>
    <zorder>widget_find2</zorder>
    <zorder>widget_gaojichazhao</zorder>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1218</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
